src/config/constants.ts

```diff
@@ -329,6 +329,8 @@ export const kCrossrefAppendixDelim = "appendix-delim";
 export const kCrossrefChaptersAlpha = "chapters-alpha";
 export const kCrossrefChapterId = "chapter-id";
 
+export const kFigResponsive = "fig-responsive";
```

Comment:
Not sure if this config should go somewhere in one of the definition interface now it is a metadata.
Currently, it is nowhere defined that this is boolean. Should it go somewhere in `types.ts` ?

Reply:
Yes, this should go in the `defaultFormat()` function. There is a bunch of other stuff you need to do for any new option -- I can check out the branch and do this once you give me the thumbs up it's okay to do

Reply:
Ok great. I tried to find the different places, but I must not have taken the correct other option as example. 

Or maybe because `defaultFormat()` does have yet an option for metadata. Anyway, I'll look at your change to learn

Reply:
ah, good point. we don't actually provide defaults for metadata values